tests/Collection/CollectionFunctionalTest.php

```diff
@@ -703,19 +705,15 @@ public function collectionReadMethodClosures(): array
     {
         return array_filter(
             $this->collectionMethodClosures(),
-            function ($rw) {
-                return str_contains($rw[1], 'r');
-            }
+            fn ($rw) => str_contains($rw[1], 'r')
```

Comment:
Since this is part of a multi-line function call, shouldn't there be a trailing comma?

Likewise for the diff immediately below.

Reply:
I can only imagine that the sniff gets tripped up by the closure, but I'll have to investigate.

Reply:
Since we're in a new class definition, `self` won't do what we expect it to do, as it will refer to the anonymous class: https://3v4l.org/GavRg